Src/GeneratedTests/Discovery.Tests/DiscoveryTest.cs

```diff
@@ -0,0 +1,76 @@
+using Google.Apis.Http;
+using Google.Apis.Services;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Threading;
+using System.Collections.Generic;
+using System.Text;
+using System.Reflection;
+using System.IO;
+
+namespace Google.Apis.Discovery.v1.Tests
+{
+    [TestFixture]
+    public class DiscoveryTest
+    {
+        class FakeHttpMessageHandler : ConfigurableMessageHandler
```

Comment:
Yes, these can certainly be moved out of here in the future, when more code might use them.